spec/js_routes/options_spec.rb

```diff
@@ -466,7 +466,9 @@
 
     context "with url_links option" do
       around(:each) do |example|
-        ActiveSupport::Deprecation.silence do
+        rails8_or_newer = Gem::Version.new(Rails.version) >= Gem::Version.new('8')
+        deprecator_object = rails8_or_newer ? Rails.deprecator : ActiveSupport::Deprecation
```

Comment:
`Rails.deprecator` introduced in `7.1`: https://github.com/rails/rails/blob/v7.1.0/railties/lib/rails/deprecator.rb


```suggestion
        Rails.version >= "7.1.0" ? Rails.deprecator : ActiveSupport::Deprecation
```